src/pages/en/guides/styling.md

```diff
@@ -268,368 +246,70 @@ Lastly, add it to your Astro page (or layout template):
 </head>
 ```
 
-As an alternative to `src/styles/global.css`, You may also add Tailwind utilities to individual `pages/*.astro` components in `<style>` tags, but be mindful of duplication! If you end up creating multiple Tailwind-managed stylesheets for your site, make sure you're not sending the same CSS to users over and over again in separate CSS files.
-
-#### Migrating from v0.19
-
-As of [version 0.20.0](https://github.com/withastro/astro/releases/tag/astro%400.20.0), Astro will no longer bundle, build and process `public/` files. Previously, we'd recommended putting your tailwind files in the `public/` directory. If you started a project with this pattern, you should move any Tailwind styles into the `src` directory and import them in your template using [Astro.resolve()][astro-resolve]:
+#### Migrating to v0.24
 
 As of [version 0.24.0-next](https://github.com/withastro/astro/releases/tag/astro%400.24.0-next.0) Astro will no longer support [Astro.resolve()][astro-resolve]. See the [Migration Guide](/en/migrate) for more details.
 
-```astro
-<link
-  rel="stylesheet"
-  href={Astro.resolve("../styles/global.css")}
->
-```
-
-### 🎭 PostCSS
-
-Using PostCSS is as simple as placing a [`postcss.config.cjs`](https://github.com/postcss/postcss#usage) file in the root of your project.
 
-Be aware that this plugin will run on all CSS in your project, including any files that compiled to CSS (like `.scss` Sass files, for example).
-
-_Note: CSS in `public/` **will not be transformed!** Instead, place it within `src/` if you’d like PostCSS to run over your styles._
+## CSS Preprocessors
 
-## Bundling
+Astro supports CSS preprocessors such as [Sass][sass], [Stylus][stylus], and [Less][less] through [Vite][vite-preprocessors].
 
-All CSS is minified and bundled automatically for you in running `astro build`. Without getting too in the weeds, the general rules are:
+### Sass
 
-- If a style only appears on one route, it’s only loaded for that route (`/_astro/[page]-[hash].css`)
-- If a style appears on multiple routes, it’s deduplicated into a `/_astro/common-[hash].css` bundle
-- All styles are hashed according to their contents (the hashes only change if the contents do!)
+ ```
+ npm install -D sass
+ ````
+ Use  `<style lang="scss">` or `<style lang="sass">` (indented) in `.astro` files
```

Comment:
What does the “indented” mean here? Should I indent `<style lang="sass">`?

Reply:
I don't know! I'm not sure where I pulled that from, but I'll check on it....